cmd/node/main.go

```diff
@@ -1052,6 +1066,32 @@ func startNode(ctx *cli.Context, log logger.Logger, version string) error {
 	return nil
 }
 
+func createStringFromRatingsData(ratingsData *rating.RatingsData) string {
```

Comment:
this should be only for a log.Debug and could be done at the constructor of NewRatingsData

Reply:
I would like it to be for all logging levels and be persistent in the folder. I can add a TODO to remove it in the future when it is not needed any more.

Reply:
added todo to remove and add just log.debug